### PR TITLE
[BenchDash] Scaffold code: Benchmark Controller and action to return benchmark list as JSON

### PR DESCRIPTION
**Pull request by tfrcarvalho**
# Description

* This PR introduces backend scaffold for retrieving benchmarks
* Defines an index action to return list of current benchmarks for the environment

# Tests

* Unit and integration tests
* To retrieve the JSON: `/benchmarks.json`

### CODE REVIEW

File: app/helpers/benchmarks_helper.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    end
+
+    return benchmarks
+  end
+
+  def benchmarks_config
+    s3_response = S3_CLIENT.get_object(bucket: IDSEQ_BENCH_BUCKET, key: IDSEQ_BENCH_KEY)
+    return JSON.parse(s3_response.body.read, symbolize_names: true)
+  rescue Aws::S3::Errors::NoSuchKey
+    LogUtil.log_err_and_airbrake("[BenchmarkController] Config file not found - '#{IDSEQ_BENCH_KEY}'")
```
Comment (jshoe) [inline app/helpers/benchmarks_helper.rb]: nit: actually BenchmarksController

Comment (tfrcarvalho) [reply app/helpers/benchmarks_helper.rb]: This was actually in the helper now! Removed the prefix.

File: app/helpers/benchmarks_helper.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
+module BenchmarksHelper
+  IDSEQ_BENCH_BUCKET = "idseq-bench".freeze
+  IDSEQ_BENCH_KEY = "config.json".freeze
+
+  def benchmarks_list
+    benchmarks = {}
+
+    # Read benchmarks from configuration store in S3
+    benchmark_config = benchmarks_config()
```
Comment (jshoe) [inline app/helpers/benchmarks_helper.rb]: this was a little confusing to read and I read it as `benchmark_config = benchmark_config` at first

Comment (tfrcarvalho) [reply app/helpers/benchmarks_helper.rb]: oh yeah... rubocop forced me to change the name and end up being the same. Renamed.

**Pull request merged**